### PR TITLE
Docs: extension common capabilities

### PR DESCRIPTION
**Pull request by jakolehm**
image

### CODE REVIEW

File: docs/extensions/capabilities/common-capabilities.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
+# Common Capabilities
+
+Common Capabilities are important building blocks for your extensions. Almost all extensions use some of these functionalities. Here is how you can take advantage of them.
+
+## Main Extension
```
Comment (jim-docker) [inline docs/extensions/capabilities/common-capabilities.md:5]: I feel there's going to be some confusion about what we call an extension. An extension (as in a package we get from the extension marketplace) consists of one or both of the extension classes/objects "Main Extension" and "Renderer Extension". What about "Main Extension Object", or "Main Extender", or something else to provide distinction?

File: docs/extensions/capabilities/common-capabilities.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+A main extension runs in the background and it's not visible directly via Lens main window. If you want to see logs from this extension you need to start Lens from the command line.
```
Comment (jim-docker) [inline docs/extensions/capabilities/common-capabilities.md]: Do you mean a main extension object does not add content directly in the UI? Maybe something like:
A main extension runs in the background and, apart from app menu items, does not add content to the Lens UI.

File: docs/extensions/capabilities/common-capabilities.md (hunk 1 above)
Comment (jakolehm) [reply docs/extensions/capabilities/common-capabilities.md:5]: I think we should clarify these in extension anatomy? @nevalla wdyt?

File: docs/extensions/capabilities/common-capabilities.md (hunk 2 above)
Comment (jakolehm) [reply docs/extensions/capabilities/common-capabilities.md]: Yes, thanks!

**Review by nevalla — APPROVED** [review]
LGTM

File: docs/extensions/capabilities/common-capabilities.md (hunk 1 above)
Comment (nevalla) [reply docs/extensions/capabilities/common-capabilities.md:5]: Yes, there are room for improvements there.

**Pull request merged**